README.md

```diff
@@ -38,17 +38,24 @@ channel](https://gitter.im/kba/hocr-spec).
 
 ## Building the spec
 
-To build the spec, you will need to have GNU make and either
-[bikeshed](https://github.com/tabatkins/bikeshed) or
-[docker](https://docker.com) installed.
+To build the spec, you will need to have GNU make and one of
+  * [bikeshed](https://github.com/tabatkins/bikeshed) or
+  * [docker](https://docker.com) or
+  * `curl` installed.
```

Comment:
OK, but I'm leaving the pc right now, will go back in a hour or two, so when I'll come back I'll do it (unless you'll do it until then).